js/5etools-backgrounds.js

```diff
@@ -424,34 +459,96 @@ function d20plusBackgrounds () {
 		let ideal = null;
 		let bond = null;
 		let flaw = null;
+		const matchCharacteristics = [
+			"Suggested Characteristics", // Most backgrounds
+			"Horror Characteristics"     // 'Haunted One' and 'Investigator'
+		];
 		// Get the JSON for all the tables
 		if (bg.entries) {
 			for (const ent of bg.entries) {
-				if (ent.name && ent.name === "Suggested Characteristics") {
+				if (ent.name && matchCharacteristics.includes(ent.name)) {
 					traits = ent;
+				} else if (ent.entries) {
+					for (const entItem of ent.entries) {
+						// look for embedded characteristics
+						if (entItem.name && entItem.name === "Suggested Characteristics") {
+							traits = entItem;
+						// look for embedded trinkets, and move to features
+						} else if (entItem.name && entItem.name.includes("Trinket")) {
+							const renderStack = [];
+							renderer.recursiveRender({entries: entItem.entries}, renderStack);
+							entItem.text = renderStack.length ? d20plus.importer.getCleanText(renderStack.join("")) : "";
+							features.push(entItem);
+						}
+					}
 				}
 			}
 		}
 
 		// Fill the rows
 		if (traits !== null && traits.entries?.length) {
 			for (let i = 0; i < traits.entries.length; i++) {
-				ent = traits.entries[i];
+				const ent = traits.entries[i];
 				// This seems to be the best way to parse the information with some room for errors
 				// It seems like the schema is based on on the website, which is why colLabels is where the identifier is
 				if (ent.colLabels && ent.colLabels.length === 2 && ent.rows) {
+
+					/**
+					 * Clean up the formatting and reference syntaxes to get clean text.
+					 *
+					 * @param entry entry to clean up
+					 * @return cleaned text
+					 */
+					const _cleanText = function (entry) {
```

Comment:
use `Renderer.stripTags( ... )` instead

Reply:
Knew something like that had to exist, but couldn't find it. Thanks!